extensions/wikia/Wall/Wall.class.php

```diff
@@ -299,7 +299,12 @@ protected function getOrderBy() {
 		}
 	}
 
-	public function getThreads( $page = 1, $master = false ) {
+	/**
+	 * @param int $page
+	 * @param bool $master
+	 * @return WallThread[]
+	 */
+	public function getThreads( int $page = 1, bool $master = false ) : array {
```

Comment:
It seems we never call this method with `$master = true`, should we remove this parameter?